projects/driver_controls_pedal/inc/steering_angle.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+// Module for steering angle angle percentage reading
+// Requires ADS1015 to be initialized
+//
+// The goal of this module is to convert the analog reading of a rotary
+// sensor to an accurate percentage output. This will be accomplished by
+// using the ADC to convert the sensor's analog input to digital input, and
+// using calibrated data from the steering angle calibration module to accurately
+// convert digital input (on a scale of 0 to 2047) to a scale of (-1024 to 1024)
+// ,which is converted to a percentage scale of (-100% to 100%),
+// where -100% is the counterclockwise-most position of the angle, and
+// 100% is the clockwise-most position of the angle.
+
```

Comment:
It looks like this code was taken from https://github.com/uw-midsun/firmware/pull/285

@ashraf2047 @rhaffar99 it was tested right?

Reply:
It was tested with a potentiometer as a substitute for the rotary sensor.